### DISCUSSION TITLE
Feature Request - capture guid page state in URL

### DISCUSSION THREAD

**Discussion by dustymc** [ArctosDB/arctos#6745] [Ideas]
**Is your feature request related to a problem? Please describe.**

Page doesn't necessarily look the same for any two users

**Describe what you're trying to accomplish** 

Allow more predictable sharing of links

**Describe the solution you'd like**

Somehow push customizations to URL?

**Describe alternatives you've considered**

don't

**Additional context**

See custom parts and (maybe eventually) identifiers views

**Priority**

?

@ewommack

**Comment by ewommack** [comment]
This was an idea I had when playing around in test with the potential new format for the identifier tables. I really liked the buttons for minimizing, expanding, or bracketing out different parts of identifiers. I thought different views might be really helpful when sharing with different collaborators or stakeholders. Perhaps the relationships vs identifiers look would be good for working with different collections, and the minimal view might be helpful for sharing with USFWS agents who just want a quick look.
But how to share these different views when the link for the specimen page might go back to the default. So maybe some way to save and share a particular view? Perhaps like the Saved Profile option in the new Data Entry page?

No particular need for this, just brain storming along.
